src/test/test-thp-disable-completely.c

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <linux/mman.h>
+#include <sys/mman.h>
+#include <sys/prctl.h>
+
+#include "tests.h"
+
+#define PR_THP_DISABLE 1
+
+static int intro(void) {
+        int res = prctl(PR_GET_THP_DISABLE, NULL, NULL, NULL, NULL);
+
+        if (res == 0)
```

Comment:
why doesn't this zero get a symbolic name?

Reply:
There isnt one in the kernel, but have added it as PR_THP_DISABLE_NOT_SET